windows-release/stage-layout-embed.yml

```
@@ -55,3 +55,26 @@ jobs:
     inputs:
       PathtoPublish: '$(Build.ArtifactStagingDirectory)\embed'
       ArtifactName: embed
+
+  - powershell: >
+      $(Python)
```

This one shouldn't need the `&` because it's been set by `layout-command.yml` (but I think we're just hoping that we never have a space in the path...)

One possibility is this (which is how `layout-command.yml` uses it), but it probably makes no difference:

```suggestion
      & "$(Python)"
```

---

Done in https://github.com/python/release-tools/pull/100/commits/c01765dacd7876de68e712362e2f8015aafae02b